Install page updates (#1223)
Qiskit/documentation: docs/start/install.mdx

      jupyter notebook path/to/notebook.ipynb
      ```

If you are planning to work locally and use simulators built into Qiskit, then your installation is done. If you want to run jobs on IBM Quantum systems, next [select an access channel](setup-channel) and finish your setup.

<Admonition type="note" title="Stay current with the latest versions">

>     Periodically run ` pip install -I` or check the [Qiskit release notes](../api/qiskit/release-notes) and the [Qiskit Runtime release notes](../api/qiskit-ibm-runtime/release-notes) so that you always have the latest versions.

Reviewer: Oh I didn't realize this is new. So usually in a project you should instead be updating your requirements.txt to the latest version so that everyone has the new version. 

```suggestion
    Periodically check the [Qiskit release notes](../api/qiskit/release-notes) and the [Qiskit Runtime release notes](../api/qiskit-ibm-runtime/release-notes) to see new releases. We recommend frequently updating your requirements for `qiskit` and `qiskit-ibm-runtime`, such as by bumping the versions in `requirements.txt`.
```

>     Periodically check the [Qiskit release notes](../api/qiskit/release-notes) and the [Qiskit Runtime release notes](../api/qiskit-ibm-runtime/release-notes) to see new releases. We recommend frequently updating your requirements for `qiskit` and `qiskit-ibm-runtime`, such as by bumping the versions in `requirements.txt`.

Author: ```suggestion
    Periodically check the [Qiskit release notes](../api/qiskit/release-notes) and the [Qiskit Runtime release notes](../api/qiskit-ibm-runtime/release-notes) to see new releases. We recommend frequently updating your requirements for `qiskit` and `qiskit-ibm-runtime` by, for example, changing the versions in `requirements.txt` to the latest versions.
```
